templates/terraform/custom_import/kms_crypto_key.go.erb

```diff
@@ -0,0 +1,24 @@
+
+	config := meta.(*Config)
+
+	// import ID is in the form `{{keyRing}}/cryptoKeys/{{name}}` and `keyRing` is in the
```

Comment:
Could you use the existing parseKmsCryptoKeyId fn here instead? That has the benefit of not requiring the project

Reply:
Will look into it. `{{project}}/{{keyring_name}}/{{key_name}}` indeed seems much simpler to specify.

Reply:
Done.